tests/basic/pure_test.go

```diff
@@ -2880,6 +2863,196 @@ var _ = Describe("{FBDAMultiTenancyBasicTest}", func() {
 	})
 })
 
+var _ = Describe("{FBDAMultiTenancyUpdatePureNFSEnpoint}", func() {
+	var contexts []*scheduler.Context
+	var customConfigAppName string
+	var origCustomAppConfigs, customAppConfigs map[string]scheduler.AppConfig
+	testName := "fbda-mt-update-endp"
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("FBDAMultiTenancyUpdatePureNFSEnpoint", "Validate Pure NFS endpoint can be changed using pxctl", nil, 0)
+		Step("setup credential necessary for cloudsnap", createCloudsnapCredential)
```

Comment:
We don't need to setup cloudsnap for Fbda tests.

Reply:
@pnookala-px actually ValidateApplicationsPureSDK() function (as part of -ve tests) invokes cloudsnaps for FBDA volumes and expect the cloudsnaps to fail with proper error saying cloudsnaps are not supported with FBDA volumes.  For this -ve test, it requires creds to be created and then cloudsnap to fail with proper error and not because of creds not present error.